doc/gambit.txi

```diff
@@ -149,6 +149,7 @@ repository.
 
 @menu
 * Accessing the system files::      Accessing the system files
+* Extending the core system::       Extending the core system
```

Comment:
This is too early in the documentation.  It is related to modules so should go in that chapter.